plugins/wdc/wdc-nvme.c

```diff
@@ -1054,6 +1054,101 @@ static int wdc_create_log_file(char *file, __u8 *drive_log_data,
 	return 0;
 }
 
+bool wdc_get_dev_mng_log_entry(
+    __u32 log_length,
```

Comment:
This project's coding conventions is to use the linux kernel style. Please use tabs instead of spaces.

Reply:
will replace spaces with tabs